backend/tournesol/views/comparison.py

```diff
@@ -116,6 +116,8 @@ def perform_create(self, serializer):
         comparison.entity_2.inner.refresh_metadata()
         comparison.entity_2.auto_remove_from_rate_later(poll=poll, user=self.request.user)
 
+        comparison.mark_compared_entities_as_seen()
```

Comment:
Note that the signal `initialize_ratings_on_comparison_creation` in "signals.py" handles creating related `ContributorRating` objects if necessary, after a new comparison is saved. 
Your version is more explicit, but modifying the signal would ensure that `entity_seen` is set even when comparisons are created outside of this specific view. Not sure that's important.

Reply:
Should be fixed by https://github.com/tournesol-app/tournesol/pull/2122/commits/1e48977ce19ef8cb55b6dcc87875fed242db8ac6